src/ansys/fluent/core/services/field_data.py

```diff
@@ -638,7 +638,56 @@ class PathlinesFieldDataRequest(NamedTuple):
     zones: list | None = None
 
 
-class TFieldData:
+class _IFieldData:
+    """The base field data interface."""
+
+    def __init__(self):
+        """__init__ method of _IFieldData class."""
+        self.data = None
+        self._field_info = None
+        self._allowed_surface_names = None
+
+    def get_surface_ids(self, surfaces: List[str | int]) -> List[int]:
```

Comment:
@prmukherj Interface classes that contain implementation can not function as interfaces. Use an alternative design to achieve this -- an intermediate base class shared by a subset of all derived class is a simple option.